projects/github-actions/repo-gardening/src/utils/openai/send-request.js

```diff
@@ -1,19 +1,19 @@
-const { getInput } = require( '@actions/core' );
+const { getInput, setFailed } = require( '@actions/core' );
 const OpenAI = require( 'openai' );
 const debug = require( '../debug' );
 
 /**
  * Send a message to OpenAI.
  *
- * @param {string} message - Message to send to OpenAI.
+ * @param {string} message        - Message to send to OpenAI.
+ * @param {string} responseFormat - Response format to use (plain by default, can be 'json_object').
  *
- * @return {Promise<string>} Promise resolving to the response from OpenAI.
+ * @return {Promise<string|void>} Promise resolving to the response from OpenAI, or void if an error occurred.
```

Comment:
JS doesn't have a "void" type. It does have a "void" operator, but that produces the "undefined" value.
```suggestion
 * @return {Promise<string|undefined>} Promise resolving to the response from OpenAI, or undefined if an error occurred.
```